### PR TITLE
Integration doc

### PR DESCRIPTION
**Pull request by redbluenat**

### CODE REVIEW

File: advanced/integrations.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
 # Integrations
 
+Mimic app is good solution for mobile developers which want to write automatic and integration tests for application. They can do this without any testing database and testing api.
```
Comment (MihalyBezzeg) [inline advanced/integrations.md:3]:
```suggestion
Mimic app is good solution for mobile developers who would like to write automatic and integration tests for their applications. They can do this without any testing database and testing API.
```

File: advanced/integrations.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
 
+In Brains and Breads we are using Detox library to to write integration test in react native apps. [More info](https://github.com/wix/Detox)
```
Comment (MihalyBezzeg) [inline advanced/integrations.md]:
```suggestion
In Brains&Breads we are using Detox library to write integration tests for React Native apps. [More info](https://github.com/wix/Detox)
```

File: advanced/integrations.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
 
+## Integration React Native App with Detox
+
+Detox is an end-to-end testing and automation library for mobile apps. It tests your mobile app while it's running in a real device/simulator, interacting with it just like a real user. We'll show you how you can use it with Mimic to test your API with mocked endpoints.
```
Comment (MihalyBezzeg) [inline advanced/integrations.md]:
```suggestion
Detox is an end-to-end testing and automation library for mobile apps. It tests your mobile app while it's running on a real device/simulator, interacting with it just like a real user. We'll show you how you can use it with Mimic to test your API with mocked endpoints.
```

File: advanced/integrations.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
 # Integrations
 
+Mimic app is good solution for mobile developers which want to write automatic and integration tests for application. They can do this without any testing database and testing api.
 
+In Brains and Breads we are using Detox library to to write integration test in react native apps. [More info](https://github.com/wix/Detox)
 
+## Integration React Native App with Detox
```
Comment (MihalyBezzeg) [inline advanced/integrations.md]:
```suggestion
## Integrating React Native Apps with Detox
```

**Comment by MihalyBezzeg** [comment]
@redbluenat I have added some suggestions for your PR. Feel free to accept / discard them 🙂

File: advanced/integrations.md (hunk 2 above)
Comment (marras) [reply advanced/integrations.md]: How about linking the [Detox library] directly rather than adding "More info" link at the end of the sentence?

**Review by marras — APPROVED** [review]
Looks good :)

**Pull request merged**